install-protobuf.sh

```diff
@@ -1,8 +1,10 @@
+#!/usr/bin/env sh
+
 set -ex
 
-basename=protobuf-2.6.0
-tarball=$basename.tar.bz2
+version=2.6.1
```

Comment:
You could easily take an arg here and default to 2.6.1:

```bash
version=${1:-"2.6.1"}
```

Thus you could install whichever version you wish without having to update the script.